ringtonepicker/src/main/java/com/kevalpatel/ringtonepicker/RingtonePickerDialog.java

```diff
@@ -354,9 +354,13 @@ public void onLoadComplete(@NonNull HashMap<String, Uri> ringtone) {
         mRingTones.putAll(ringtone);
 
         final String[] itemTitles = mRingTones.keySet().toArray(new String[mRingTones.size()]);
+        int currentSelectionPos = getUriPosition(mRingTones, mCurrentRingTone.second);
+
         mListView.setAdapter(new ArrayAdapter<>(mContext,
-                android.R.layout.select_dialog_singlechoice,
+                android.R.layout.simple_list_item_single_choice,
```

Comment:
Why do you change the layout?

Reply:
I needed this layout in my code :)
Not of major concern i guess.

Reply:
Please revert it to the old `android.R.layout.select_dialog_singlechoice` and after this, I will merge this PR. Everything else looks fine.